.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: ci
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Java ${{ matrix.java }}
+    strategy:
+      matrix:
+        java: [8, 11]
```

Comment:
Looks like these are OpenJDK versions, meaning we lose the OracleJDK from the previous matrix. This shouldn't be a problem, but noting in case someone else knows if we'd need it.

Reply:
I don't think it's a problem.